source/grid/grid_tools.cc

```diff
@@ -4191,14 +4191,52 @@ namespace GridTools
     const std::vector<Point<spacedim>> &points,
     const typename Triangulation<dim, spacedim>::active_cell_iterator
       &cell_hint)
+  {
+    const auto cqmp = compute_point_locations_try_all(cache, points, cell_hint);
+    // Splitting the tuple's components
+    const auto &cells    = std::get<0>(cqmp);
+    const auto &qpoints  = std::get<1>(cqmp);
+    const auto &maps     = std::get<2>(cqmp);
+    const auto &n_points = std::get<3>(cqmp);
```

Comment:
this is actually a vector, so maybe `n_point` is the wrong kind of name for the variable?

Reply:
I guess so, I renamed it `missing_points`; is it ok?